gallery/src/main.scss

```diff
@@ -81,3 +81,12 @@
     display: none;
   }
 }
+
+#temporary-navigation-sidebar-example {
+  .custom-tree-content {
+    font-size: $nx-font-size-s;
+    * {
+      color: $nx-white;
+    }
+  }
+}
```

Comment:
Missing final newline.

Reply:
> Missing final newline.

Added in fc21a6cb2ff425affed5d5c8d0002198dd968185